src/atomate2/qchem/sets/base.py

```diff
@@ -0,0 +1,511 @@
+                    for k, v in temp_plots.items():
+                        plots_dict[k] = v
+                if sub == "nbo":
+                    if nbo_dict is None:
+                        raise RuntimeError(
+                            "Can't overwrite nbo params when NBO is not being run! Exiting..."
+                        )
+                    temp_nbo = lower_and_check_unique(sub_dict)
+                    for k, v in temp_nbo.items():
+                        nbo_dict[k] = v
+                if sub == "geom_opt":
+                    if geom_opt_dict is None:
+                        raise RuntimeError(
+                            "Can't overwrite geom_opt params when not using the new optimizer! Exiting..."
+                        )
+                    temp_geomopt = lower_and_check_unique(sub_dict)
+                    for k, v in temp_geomopt.items():
+                        geom_opt_dict[k] = v
+                if sub == "opt":
+                    temp_opts = lower_and_check_unique(sub_dict)
+                    for k, v in temp_opts.items():
+                        opt_dict[k] = v
+
+            self.rem_dict = rem_dict
+            self.opt_dict = opt_dict
+            self.pcm_dict = pcm_dict
+            self.solv_dict = solv_dict
+            self.smx_dict = smx_dict
+            self.scan_dict = scan_dict
+            self.vdw_dict = vdw_dict
+            self.plots_dict = plots_dict
+            self.nbo_dict = nbo_dict
+            self.geom_opt_dict = geom_opt_dict
+
+    def get_input_set(
+        self,
+        molecule: Molecule = None,
+        # prev_dir: Union[str, Path] = None,
+        overwrite_inputs: dict = None,
+    ) -> QChemInputSet:
+        """
+        Get a QChem Input Set as a dictionary for a molecule
+
+        Parameters
+        ----------
+        molecule
+            A pymatgen molecule.
+        prev_dir
+            A previous directory to generate the input set from.
+
+        Returns
+        -------
+        QchemInputSet
+            A QChem input set
+        """
+
+        # molecule, prev_basis, prev_scf, new_geom_opt, overwrite_inputs, nbo_params = self._get_previous(
+        #     molecule, prev_dir
+        # )
+
+        # basis_set_updates = self.get_basis_set_updates(
+        #     molecule,
+        #     prev_basis=prev_basis,
+        #     prev_scf=prev_scf,
+        #     new_geom_opt=new_geom_opt,
+        #     overwrite_inputs=overwrite_inputs,
+        #     nbo_params=nbo_params,
+        # )
+
+        # scf_algorithm_updates = self.get_scf_algorithm_updates(
+        #     molecule,
+        #     prev_basis=prev_basis,
+        #     prev_scf=prev_scf,
+        #     new_geom_opt=new_geom_opt,
+        #     overwrite_inputs=overwrite_inputs,
+        #     nbo_params=nbo_params,
+        # )
+
+        # basis_set = self._basis_set(basis_set_updates)
+        # scf_algorithm = self._get_scf_algorithm(scf_algorithm_updates)
```

Comment:
Reminder that from previous and `get_input_set_updates` are not implemented.

Reply:
Thanks a lot @utf for reviewing this! I am happy that we are moving closer :-) . I will be out for a conference next week. I hope to get back to this and implement all of what's left sometime in April!

I will ping you for feedback and assistance.

Reply:
After discussions with @rkingsbury , this whole paradigm has been moved out for now